Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/builtin_http_server.py

```diff
@@ -38,21 +41,50 @@ def do_GET(self):
         self.server_shutdown()
 
     def do_POST(self):
```

Comment:
We may need to duplicate this logic in `twisted_http_server` if we think http can fit our needs for signpost.

Reply:
What is `twisted_http_server` used for?
(I see the .py file and it is listed in the README.md,  but I don't see it being used anywhere...
https://searchfox.org/wubkat/search?q=twisted_http_server&path=&case=false&regexp=false

Reply:
It's another server serving the same purpose as this file. However, there are some cases current http server does not work but works with twisted server.